### DISCUSSION TITLE
color change html buttons are not visible in AR mode

### DISCUSSION THREAD

**Discussion by Arumugamsix** [google/model-viewer#2618] [Q&A]
**Description**

We have done a car configurator it includes a color change module and it's working fine in normal mode. but in AR mode the color buttons are not visible. but our requirement is change the color in AR mode also

**working site**
https://hnsonline.com/test/

**Comment by elalish** [comment]
Can you show a screen recording so I can tell exactly what you mean? I notice a few things: 
1) Check your console logs; it looks like you're importing two different versions of the model-viewer library. Make sure you're only using the latest.
2) HTML child element will only show up in WebXR AR mode (Scene Viewer and Quick Look don't support them). However, I see you're using `ios-src`. With our latest version you can remove this and we'll auto-generate the USDZ for you, which should load the currently-selected color into Quick Look (it's faster too). 
3) Look closely at our [docs](https://modelviewer.dev/docs/#augmentedreality); for instance I notice you have `ar-scale: "30%"`, which is not a supported value.
